client/src/app/site/motions/modules/motion-detail/components/motion-detail/motion-detail.component.ts

```diff
@@ -503,6 +508,10 @@ export class MotionDetailComponent extends BaseModelContextComponent implements
 
     public enterEditMotion(): void {
         this.editMotion = true;
+
+        if (this.editMotion) {
+            this.showMotionEditConflictWarningIfNecessary();
+        }
```

Comment:
In which case is this `false`?
I think, it can only be `this.showMotionEditConflictWarningIfNecessary()` (without a condition).

Reply:
* you click "edit motion"
* if you happen to enter edit mode 
* `showMotionEditConflictWarningIfNecessary` detects if your motion has any paragraph based amendments
* if does it shows a warning, that editing that motion will lead into inconsistencies

Reply:
Hmm, I meant the condition, you check:
```ts
if (this.editMotion) {
    this.showMotionEditConflictWarningIfNecessary();
}
```
I don't know, if you are referring to this. I think, it can only be:
```ts
this.showMotionEditConflictWarningIfNecessary();
```
It's because two lines above, the flag `editMotion` will set to true (every time, someone enters the edit mode).

Reply:
ah yes, have not seen that